src/json/include/sourcemeta/jsontoolkit/json_value.h

```diff
@@ -861,7 +861,8 @@ class SOURCEMETA_JSONTOOLKIT_JSON_EXPORT JSON {
   /// const auto result = document.try_at("foo");
   /// EXPECT_TRUE(result.has_value());
   /// EXPECT_EQ(result.value().to_integer(), 1);
```

Comment:
```suggestion
  /// EXPECT_EQ(result.value().get().to_integer(), 1);
```

The example needs to be updated